x/evm/keeper/keeper.go

```diff
@@ -117,24 +121,6 @@ func (k Keeper) SetBlockBloom(ctx sdk.Context, height int64, bloom ethtypes.Bloo
 	store.Set(key, bloom.Bytes())
 }
 
-// GetBlockBloomTransient returns bloom bytes for the current block height
-func (k Keeper) GetBlockBloomTransient() (*big.Int, bool) {
-	store := k.ctx.TransientStore(k.transientKey)
-	bz := store.Get(types.KeyPrefixTransientBloom)
-	if len(bz) == 0 {
-		return nil, false
-	}
-
-	return new(big.Int).SetBytes(bz), true
-}
-
-// SetBlockBloomTransient sets the given bloom bytes to the transient store. This value is reset on
-// every block.
-func (k Keeper) SetBlockBloomTransient(bloom *big.Int) {
-	store := k.ctx.TransientStore(k.transientKey)
-	store.Set(types.KeyPrefixTransientBloom, bloom.Bytes())
-}
-
```

Comment:
why get rid of the transient store for the bloom filter?

Reply:
I'm not sure what's the difference between transient store and in memory directly.

Reply:
done, changed to transient store.

Reply:
Ref: https://docs.cosmos.network/master/core/store.html#transient-store